gz-plugins/nvme_diskio.prom

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Copyright 2025 Edgecast Cloud LLC.
+
+set -o errexit
+set -o pipefail
+
+# shellcheck disable=SC2154
+if [[ -n "$TRACE" ]]; then
+    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+    set -o xtrace
+fi
+
+# Get list of disk devices from diskinfo
+# This returns only NVMe disks with format: TYPE DISK VENDOR MODEL SIZE REMOVABLE SOLID_STATE
```

Comment:
Format the above comment to be 80-column happy, please?

Reply:
Fixed here: https://github.com/TritonDataCenter/triton-cmon-plugins/pull/2/commits/a7bd0bba76b2165352f9736f8b2bddb7ef103384